### PR TITLE
Sequence VE enhancement

### PR DESCRIPTION
**Pull request by yangli0516**
A few enhancements on the sequence VE.

### CODE REVIEW

**Comment by stevewh** [comment]
Had a quick look and this seems like a great improvement to the structure
UI. I will apply the diff locally to test it on several different READ
versions.

Andrew, let me know what you think.

File: editors/js/sequenceVE.js [hunk 1]
```diff
@@ -901,16 +901,25 @@ EDITORS.SequenceVE.prototype = {
...
-* @param string array entGIDs Entity global identifiers\n* @param cntSubLevels
+* @param string array entGIDs Entity global identifiers
+* @param cntSubLevels
+* @param {boolean} isRoot Whether the current level is at the root level.
 *
 * @returns {Array}
 */
 
-  getSubItems: function(entGIDs, cntSubLevels) {
+  getSubItems: function(entGIDs, cntSubLevels, isRoot) {
```
Comment (xadxura) [inline editors/js/sequenceVE.js:932]: isRoot should be an optional parameter.

**Review by xadxura — APPROVED** [review]
Looks good to me. Cheers.

**Comment by stevewh** [comment]
Ok. Since Ian said that they are on vacation, I will run tests on local
versions before the new year when I can. Then we can pull.

**Comment by stevewh** [comment]
I tested and it does seem much better and stable for the control.  I did have an issue that will need to be checked when I ran this on an old LMU DB that had existing structure. I was able to move a chapter and was not able to move it back or directly move it above or below it's siblings
image

**Pull request merged**